### DISCUSSION TITLE
PDF Conversion from iPython

### DISCUSSION THREAD

**Discussion by chulminy** [chulminy/CIVE497-CIVE700#157] [General]
All

Can anyone provide instructions on how to convert IPython notebooks to PDF? One of our students is struggling with this, and it would be greatly appreciated if you could share your method

Chul Min

**Comment by SMRazaRizvi96** [comment]
I use Jupyter Notebooks in VSCode and exporting to PDF is simple there.

**Step 1:**

Follow this link to set-up Jupyter Notebooks in VSCode. The tutorial leads to creating a new notebook, which can be skipped and the Notebooks given in the Tasks can be directly opened after the setup.

https://www.youtube.com/watch?v=h1sAzPojKMg

**Step 2:**

Now, to export it into PDF, there is a built-in command that comes with the Jupyter Notebook plugin in VSCode, but that requires a complex setup.
A simple way is to export to HTML and then convert the HTML page to PDF using Chrome > Print ...

In order to export into HTML, click on the three dots on top of the Jupyter Notebook in VSCode > Export > HTML.

I hope this helps.
